.github/workflows/test.yml

```diff
@@ -147,11 +152,12 @@ jobs:
       run: ./bash/docker_container_setup.sh
 
     - name: Pytest
+      env:
+        calls: ${{ toJSON(matrix.calls) }}
       run: |
-        echo '${{ toJson(matrix.calls) }}' \
-          | sed '$ d;1d;s/^[[:space:]]*\"//;s/\",\{0,1\}[[:space:]]*$//'\
-          | while read call; do
-          python3 -m pytest $call
+        echo "$calls" | jq -cr ".[]" | while read call; do
+          IFS=$'\n' arr=( $(xargs -n1 <<<"$call") )
```

Comment:
Couldn't get `readarray` to work as I wanted but `xargs` was simpler @alexrichey

Reply:
Hope we never think about that monstrous sed call I initially wrote again. It worked! But so does the tiny `jq` call it's replaced with